### PR TITLE
Feature/pure manifest integration

### PR DESCRIPTION
**Pull request by thallesfreitaszup**

### CODE REVIEW

File: test/module/module_suite_test.go [hunk 1]
```diff
@@ -0,0 +1,27 @@
+// Copyright 2022 ZUP IT SERVICOS EM TECNOLOGIA E INOVACAO SA
```
Comment (tiagoangelozup) [inline test/module/module_suite_test.go:1]: Put the file in the same package as the one being tested.
e.g. `pkg/module/manifest_validation_test.go`

File: config/flux/patch.yaml [hunk 2]
```diff
@@ -19,4 +19,4 @@
   path: "/spec/template/spec/containers/0/args/5"
 - op: add
   path: "/spec/template/spec/containers/0/args/-"
-  value: --storage-adv-addr=charlescd-flux-source-controller.$(RUNTIME_NAMESPACE).svc.cluster.local.
+  value: --storage-adv-addr=flux-source-controller.$(RUNTIME_NAMESPACE).svc.cluster.local.
```
[the remark this answers is not in the document]
Comment (thallesfreitaszup) [reply config/flux/patch.yaml]: removed

File: test/module/module_suite_test.go (hunk 1 above)
Comment (thallesfreitaszup) [reply test/module/module_suite_test.go]: done!

File: internal/client/manifest.go [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+	"github.com/hashicorp/go-getter"
+)
+
+type Manifest struct{}
+
+func NewManifest(manifests ManifestsReader) *Manifest {
+	return &Manifest{}
+}
+
+func (h *Manifest) DownloadFromSource(ctx context.Context, source, path string) (string, error) {
```
[the remark this answers is not in the document]
Comment (thallesfreitaszup) [reply internal/client/manifest.go]: done!

**Review by tiagoangelozup — APPROVED** [review]

**Review by nathannascimentozup — APPROVED** [review]

**Review by nathannascimentozup — APPROVED** [review]

**Review by nathannascimentozup — DISMISSED** [review]

**Review by nathannascimentozup — APPROVED** [review]

**Review by tiagoangelozup — APPROVED** [review]

**Pull request merged**